clients/app/src/core/stores/footballClubs/index.ts

```diff
@@ -13,9 +15,10 @@ const footballClubStore = generateCRUD<TeamViewModel>({
     });
   },
 
-  loadById: (id: DbIdentifier) => {
-    return sdk.getTeam(id);
-  },
+  // @NOTICE: will rise compilation error due to only 'list' passed as KeyType.
```

Comment:
Why? cant we just ignore method if its key is not provided? later we may enhance this, and maybe keys will not be required, we may parse key from method name, to minimize developer experience

Reply:
And maybe the name `loadById` need to be renamed to `loadSingle` ? in general?

Reply:
do you mean if method 'loadSingle' was provided then we can use single sate?

Reply:
Yep, but this a is raw idea, but renaming function is for now